CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ BUG FIXES:
 
 * cli: Fix missing identifying attributes (e.g. "id", "name") when displaying plan diffs with nested objects. [GH-30685]
 * functions: Fix error when `sum()` function is called with a collection of string-encoded numbers, such as `sum(["1", "2", "3"])`. [GH-30684]
+* Terraform will no longer crash with "invalid memory address or nil pointer dereference" error during a `terraform apply` command. ([#30780](https://github.com/hashicorp/terraform/pull/30780))
```

Comment:
This claim is a little too broad to be useful to readers of the changelog—there are several possible sources of a nil pointer dereference, and we've only fixed one of them. Can you be more specific about the context in which the error that you've fixed would happen?